### PR TITLE
Match test_item_entity_position

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.01%) on `us`.

File: src/C50A0.c [hunk 1]
```diff
@@ -258,7 +258,53 @@ void func_80133A94(s32 idx, s32 itemID) {
...
+    f32 dx, dy, dz;
+    s32 flags;
+    s8 type;
+    s32 i;
+
+    if (is_starting_conversation() || D_801565A4 || get_time_freeze_mode() || gOverrideFlags & GLOBAL_OVERRIDES_CANT_PICK_UP_ITEMS) {
+        return -1;
+    }
+
+    for (i = 0; i < MAX_ITEM_ENTITIES; i++){
```
Comment (ethteck) [inline src/C50A0.c:272]: not at my dev environment, but can you just do ARRAY_COUNT(D_801565A0) ? I think that's better than the constant here

File: src/C50A0.c [hunk 2]
```diff
@@ -258,7 +258,53 @@ void func_80133A94(s32 idx, s32 itemID) {
 s32 test_item_player_collision(ItemEntity* itemEntity);
 INCLUDE_ASM(s32, "C50A0", test_item_player_collision);
 
-INCLUDE_ASM(s32, "C50A0", test_item_entity_position);
+s32 test_item_entity_position(f32 x, f32 y, f32 z, f32 dist) {
+    ItemEntity* item;
+    f32 dx, dy, dz;
+    s32 flags;
```
Comment (ethteck) [inline src/C50A0.c:264]: Flags and type temps seem unused and can be removed I think

**Review by ethteck — CHANGES_REQUESTED** [review]
Nice!

**Review by nanaian — APPROVED** [review]

File: src/C50A0.c (hunk 1 above)
Comment (JoshDuMan) [reply src/C50A0.c:270]: No, you can't.

**Pull request merged**